rocks-bin/src/check.rs

```diff
@@ -0,0 +1,38 @@
+use eyre::{OptionExt, Result};
+use rocks_lib::{
+    build::BuildBehaviour,
+    config::Config,
+    lockfile::PinnedState::Pinned,
+    manifest::Manifest,
+    operations::{self, install},
+    progress::MultiProgress,
+    project::Project,
+};
+
+pub async fn check(config: Config) -> Result<()> {
+    let project = Project::current()?.ok_or_eyre("Not in a project!")?;
+
+    let src_dir = project.src_dir().ok_or_eyre(
+        "No source directory found! Expected a `src/` or `lua/` directory at the project root.",
+    )?;
+
+    let manifest = Manifest::from_config(config.server(), &config).await?;
```

Comment:
After #285, this should be a `RemotePackageDB`.

Reply:
Yeah this was a rebase mistake on my end.